Sources/MoonFoundation/Publisher+Extensions.swift

```diff
@@ -0,0 +1,27 @@
+import Combine
+
+public extension Publisher {
+    func sinkValue(_ receiveValue: @escaping ((Self.Output) -> Void)) -> AnyCancellable {
+        sink(
+            receiveCompletion: { _ in },
+            receiveValue: receiveValue
+        )
+    }
+
+    func mapTo() -> Publishers.Map<Self, Void> {
```

Comment:
Would it be better to name this method mapToVoid()?

Reply:
not sure if it will be better, let's proceed with just `mapTo` for now

Reply:
For the reference `void()` would also work.